src/Documentation/DocumentationGenerator.php

```diff
@@ -171,31 +171,26 @@ public function generateFixerDocumentation(FixerInterface $fixer): string
             }
         }
 
-        $riskyDescription = null;
-        $samples = [];
+        $definition = $fixer->getDefinition();
 
-        if ($fixer instanceof FixerInterface) {
```

Comment:
The whole fix here is simply removing this `if` condition as `$fixer` is type-hinted to be `FixerInterface`.

Reply:
hehe, i remember we were introducing the `FixerInterface` and not each of 200+ rules were yet following it. good old times